### DISCUSSION TITLE
Welcome to MicrosoftAzureAI Discussions!

### DISCUSSION THREAD

**Discussion by Daniel-Krzyczkowski** [Daniel-Krzyczkowski/AzureAI#2] [General]
Do not hesitate to ask any questions you have about this project!

**Comment by AntonioChiancone** [comment]
Very interesting case. Is it possible to have azure resource creation templates?

**Reply by Daniel-Krzyczkowski** [reply]
Thank you. Unfortunately, I do not have ARM templates.
